src/main/java/com/feedzai/commons/sql/abstraction/engine/AbstractDatabaseEngine.java

```diff
@@ -221,6 +222,14 @@ protected String getPrivateKey() throws Exception {
         return readString(new FileInputStream(f)).trim();
     }
 
+    /**
```

Comment:
WDYT of having this instead:
```
Get the properties to be passed to the database connection that will be created. Since these properties are specific to the database engine, it is up to the {@link AbstractDatabaseEngine} implementation to specify them. This is used, for instance, to set the database connection timeout.
```

Reply:
Yes, that's much better. Thanks for the feedback.